build.html

```diff
@@ -1,42 +1,42 @@
 <div class="passcode-wrapper" data-lock-id="{{id}}" data-lock-uuid="{{uuid}}">
   <div class="state" data-state="setup">
-    <h1>Set up a Passcode</h1>
-    <input type="tel" class="passcode-field num-setup" placeholder="Tap to enter passcode" />
-    <p>To protect your data and unlock your app you will need to set up a 4-digit passcode</p>
-    <p class="text-danger">Passcodes did not match. Please try again.</p>
+    <h1 data-translate="widgets.lock.setup.title">Set up a Passcode</h1>
+    <input type="tel" class="passcode-field num-setup" data-translate="[placeholder]widgets.lock.setup.passwordPlaceholder" placeholder="Tap to enter passcode" />
+    <p data-translate="widgets.lock.setup.instruction">To protect your data and unlock your app you will need to set up a 4-digit passcode</p>
+    <p class="text-danger" data-translate="widgets.lock.setup.mismatch">Passcodes did not match. Please try again.</p>
   </div>
   <div class="state future" data-state="verify">
-    <h1>Verify your Passcode</h1>
+    <h1 data-translate="widgets.lock.verify.title">Verify your Passcode</h1>
     <div class="verify-field-wrapper">
-      <input type="tel" class="passcode-field num-verify" placeholder="Tap to re-enter passcode" />
+      <input type="tel" class="passcode-field num-verify" data-translate="[placeholder]widgets.lock.verify.passwordPlaceholder" placeholder="Tap to re-enter passcode" />
       <div class="back-setup"><span class="fa fa-chevron-left"></span></div>
     </div>
-    <p>To protect your data and unlock your app you will need to set up a 4-digit passcode</p>
+    <p data-translate="widgets.lock.verify.instruction">To protect your data and unlock your app you will need to set up a 4-digit passcode</p>
   </div>
   <div class="state future" data-state="touchID">
-    <h1>Complete</h1>
-    <p>Your passcode is successfully set up.</p>
-    <p>You can also use <span class="bio-id-type">Face ID</span> to unlock your app in the future.</p>
-    <button class="btn btn-primary continue-touchID">Continue</button>
+    <h1 data-translate="widgets.lock.touchId.title">Complete</h1>
+    <p data-translate="widgets.lock.touchID.success">Your passcode is successfully set up.</p>
```

Comment:
This shows the changes weren't tested after they are changed from `touchID` to `touchId`. Please show screencast of all the messages working.

```suggestion
    <p data-translate="widgets.lock.touchId.success">Your passcode is successfully set up.</p>
```

Reply:
> This shows the changes weren't tested after they are changed from `touchID` to `touchId`. Please show screencast of all the messages working.

Yes, I'm sorry. I missed it.